studio/components/interfaces/Organization/TeamSettings/TeamSettings.tsx

```diff
@@ -89,9 +89,10 @@ const TeamSettings = () => {
             placeholder="Filter members"
           />
           <ScaffoldActionsGroup>
-            {canAddMembers && profile !== undefined && (
+            {canAddMembers && profile !== undefined && selectedOrganization !== undefined && (
```

Comment:
`... && profile !== undefined && selectedOrganization !== undefined && ... `

this seems like it should be part of the permission check.
more things to go wrong?

Reply:
future consideration is perhaps the permissions check hook actually returns the message to show as well? 

There's a lot of interfaces having to handle the permission message.

for instance.. if no profile the message would be along the lines of:
"No profile detected"

might be a bit overkill though.

Reply:
ah cool. no worries then